Some corrections and additions

CirKu17 opened the pull request:
Based on  https://github.com/evilsocket/Telegram/blob/f3b0d976ae371d1270b5ab86e2781360b7332838/TMessagesProj/src/main/res/values-it/strings.xml .
Hoping this repo is the right one.

CirKu17 committed:
Some corrections and additions

Based on https://github.com/evilsocket/Telegram/blob/f3b0d976ae371d1270b5ab86e2781360b7332838/TMessagesProj/src/main/res/values-it/strings.xml

AlexGreg commented:
@mikfaina @mlocati this PR seems more complete

mlocati commented:
Yes, but with some bad rules.
As a reference doc I usually use this one: http://tp.linux.it/buona_traduzione.html

So, instead of `Nome Nascosto` I'd use `Nome nascosto` (and so on...).
Furthermore, instead of normal single quotes `'` I'd use their typographic variant `’`…

AlexGreg commented:
that typographic variant, in my case at least, it's causing compile problems. I didn't try to backspace it though
the choice for using lower case, I agree

mlocati commented:
@AlexGreg did you saved the file in utf8 (without bom)?

mlocati commented:
@CirKu17 l'italiano e l'inglese sono due lingue diverse, con regole e stili diversi.... se leggi il documento che ho linkato la traduzione sarà migliore e con uno stile più simile a tutti gli altri programmi

CirKu17 commented:
(Chiuso per sbaglio)

CirKu17 committed:
Correction

As in https://github.com/CirKu17/Telegram/commit/6355d18a81a031d6a825f1d14da01acbb8707a7f#tmessagesproj-src-main-res-values-it-strings-xml-P288

CirKu17 committed:
Adaptation of capitalization

as proposed by @mlocati

CirKu17 committed:
Fix of quotation marks escape

AlexGreg commented:
:+1:

mlocati commented:
Sei sicuro che si debba scrivere `\’` invece di `’`? Mi sembra così strano...

CirKu17 commented:
@mlocati Ho considerato queste guideline:
https://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling
E altre occasioni sono state escapate allo stesso modo.

mlocati commented:
Sì, `\` va messo prima di `'`, ma `’` è un altro carattere...

CirKu17 commented:
@mlocati  eh si, purtroppo ci sono occasioni di entrambi i caratteri, e nella ricerca sono considerati uguali per qualche motivo. Meglio riportare tutto a `\'`  o a `’` ?

mlocati commented:
Direi che `’` è meglio: in teoria è il carattere tipografico che andrebbe usato (vedi per esempio http://it.wikipedia.org/wiki/Apostrofo), e `’` è più leggibile di `\'`

AlexGreg commented:
non saprei..anche su Wikipedia ti mettono l'immagine dell'apostrofo stilato, ma nello stesso articolo viene usato quello semplice.
In più le altre lingue usano <code>'</code> (vedi inglese)

Io opterei per <code>\'</code>

mlocati commented:
Il fatto è che `’` non è presente sulla tastiera, ergo chi scrive (anche Italiani) di solito scrivono `'`.
Per le altre lingue... Ogni lingua fa da se.

Ad ogni modo non è un problema così grave :wink:

AlexGreg commented:
In teoria è presente <code>AltGr+'</code>

mlocati commented:
Su Windows occorre fare `Alt`+`0146` sul tastierino numerico... Non proprio di facile accesso...

AlexGreg commented:
uh..infatti mi riferivo su un SO Linux..non pensavo che su Windows fosse disabilitata questa scelta rapida..
Comunque come hai detto non è un problema grave
Riguardo le altre stringhe non ancora presenti invece?

mlocati commented:
Per la gestione delle stringhe non vedo l'ora che si passi a Transifex (https://github.com/DrKLO/Telegram/issues/127): gestirle a mano e non collaborativamente è un casino...
Ho creato un progetto di prova, lo trovi qui: [RIMOSSO]

PS: le stringhe di riferimento in inglese vengono aggiornate automaticamente una volta al giorno

mlocati commented:
Credo che @DrKLO abbia già creato il progetto su Transifex (ieri): https://www.transifex.com/projects/p/telegram/

CirKu17 commented:
Faccio un commit passando tutto a `\'` intanto, che mi sembra più standard. Su transifex si vedrà.

CirKu17 committed:
Uniforming quotation mark symbols

Converting `’` to `\'`
